recipe_scrapers/primaledgehealth.py

```diff
@@ -20,7 +21,22 @@ def image(self):
         return self.schema.image()
 
     def ingredients(self):
-        return self.schema.ingredients()
+        ingredients_li = self.soup.select(".wprm-recipe-ingredient")
+        ingredients_list = [
+            normalize_string(
+                li.find("span", {"class": "wprm-recipe-ingredient-amount"}).get_text()
+            ).replace("\u00C2", "")
+            + " "
+            + normalize_string(
+                li.find("span", {"class": "wprm-recipe-ingredient-unit"}).get_text()
+            ).replace("\u00C2", "")
+            + " "
+            + normalize_string(
+                li.find("span", {"class": "wprm-recipe-ingredient-name"}).get_text()
+            ).replace("\u00C2", "")
+            for li in ingredients_li
+        ]
+        return ingredients_list
```

Comment:
```suggestion
        ingredients = []
        container = self.soup.select(".wprm-recipe-ingredient")
        for item in container:
            amount, unit, name = (
                item.find("span", {"class": "wprm-recipe-ingredient-amount"}),
                item.find("span", {"class": "wprm-recipe-ingredient-unit"}),
                item.find("span", {"class": "wprm-recipe-ingredient-name"}),
            )
            ingredients.append(f"{amount.text} {unit.text} {name.text}")
        return [
            normalize_string(ingredient).replace("\u00C2", "")
            for ingredient in ingredients
        ]
```

Reply:
Would it be possible to simplify that loop further:

```python
for item in container:
    ingredients.append(item.get_text())
```

instead of handling the quantity, unit and name separately, then combining.

Reply:
Almost: there's some additional text that may or may not be useful to filter out (`optional, see note` in one of the items here).

Reply:
In fact, I think we should probably prefer to use the `schema.org` metadata on the page here.  It does contain the unexpected/probably-out-of-place `\u00C2` character too, but we can retain the `.replace` fixup to handle that.  I've updated the code suggestion accordingly.